queries.sql

```diff
@@ -23,3 +23,29 @@ SELECT * from animals WHERE NOT name='Gabumon';
 
 -- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg).
 SELECT * from animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+
+
+
+-- Update and Querry for the animal Table
+
+-- How many animals are there?
+SELECT COUNT(name) from animals;
+
+-- How many animals have never tried to escape?
+SELECT COUNT(name) from animals WHERE escape_attempts = 0;
+
+-- What is the average weight of animals?
+SELECT AVG(weight_kg) from animals;
+
+-- Who escapes the most, neutered or not neutered animals?
+SELECT name, escape_attempts from animals
+WHERE escape_attempts = (select MAX(escape_attempts) from animals);
+
+-- What is the minimum and maximum weight of each type of animal?
+SELECT specie, MAX(weight_kg), MIN(weight_kg) from animals GROUP BY specie;
+
+-- What is the average number of escape attempts per animal type of those born between 1990 and 2000?
+SELECT specie, AVG(escape_attempts) FROM animals
+WHERE date_of_birth >= '1990-01-01' AND date_of_birth < '2000-01-01'
```

Comment:
Please note, it's where date of birth <= 2000/31/12, i.e last day of the last month in year 2000

- [ ] Kindly update it